src/main/java/seedu/duke/event/Event.java

```diff
@@ -0,0 +1,29 @@
+package seedu.duke.event;
+
+import seedu.duke.company.CompanyList;
+import seedu.duke.exception.InvalidIndexException;
+import seedu.duke.venue.Venue;
+import seedu.duke.venue.VenueList;
+
+public class Event {
+
+    private Venue venue;
+
+    private CompanyList companyList;
+
+    public Event(CompanyList companyList){
```

Comment:
perhaps a space can be added between `)` and `{` for the codestyle to be consistent throughout